controllers/account.js

```diff
@@ -47,6 +47,7 @@ router.post('/', (req, res) => {
 router.get('/', (req, res, next) => {
 	const isSSO = Boolean(res.locals.currentPayload.systemId);
 	const isDiscoverable = res.locals.currentUser.discoverable;
+	const hideVisibilitySettings = (res.locals.currentRole === 'Schüler' || res.locals.theme.name === 'n21');
```

Comment:
use process.env.IGNORE_DISCOVERABILITY which is used in the server already instead of res.locals.theme.name === 'n21'